website/src/components/Survey/LabelLikertGroup.tsx

```diff
@@ -18,160 +19,154 @@ interface LabelInfo {
   inverted: boolean;
 }
 
-const getLabelInfo = (label: string): LabelInfo => {
+const getLabelInfo = (label: string, t: (key: string) => string): LabelInfo => {
```

Comment:
This is not typesafe? Could you look more into it? Mabe the following will work
`t: ReturnType<typeof useTranslation>["t"]`

Reply:
I've changed it. I don't know typescript, so I don't know if it's right. It seems to work.

Reply:
It looks like the E2E tests have failed. I've taken a look, but I don't understand why, sorry.

Reply:
@notmd I've been working on this. I've found that the solution you proposed causes an infinite loop in the type checking and that was the cause of the E2E test failing.

I've gone back to my initial solution. I think that it is type safe. As I understand it, it's saying that parameter `t` is a function that receives a `string` and returns a `string`. Isn't that correct?

Reply:
So I'm fine with the current behavior. Thanks